src/resources.c

```diff
@@ -738,7 +738,13 @@ content_t* res_file_content(const void *res, hash_t *facts)
 		rewind(runner.in);
 
 		errno = 0;
-		int rc = run2(&runner, "cw", "template-erb", r->template, NULL);
+		int rc;
+		if (strlen(r->template) > 4
+				&& strcmp(r->template + strlen(r->template) - 4, ".erb") == 0) {
+			rc = run2(&runner, "cw", "template-erb", r->template, NULL);
+		} else {
+			rc = run2(&runner, "cw", "template-tt",  r->template, NULL);
+		}
```

Comment:
This breaks backwards-compatibility for people who are not using `.erb` suffixes.  Can we flip the disposition around so that `.tt` engages `cw-template-tt`?  While we're at it, I would totally be okay with setting a default template format configuration value somewhere, and then aligning the `.erb` suffix with `cw-template-erb`, followed by `cw-template-${new_default_template_format}` for unmatched file names.